### ISSUE TITLE
7.7.x UI Regression: "Closed" and "All" buttons under Milestones for project group don't work

### ISSUE THREAD

**Issue by exabrial** [gitlabhq/gitlabhq#8751]
The "Closed" and "All" filter buttons on the Milestone view for a project group don't work. If you navigate to the the following example links, open milestones are always displayed:

* http://gitlab/groups/my-group/milestones?state=all
* http://gitlab/groups/my-group/milestones?state=closed

**Comment by exabrial** [comment]
No errors in the log :(

```
Started GET "/groups/my-group/milestones?state=closed" for 127.0.0.1 at 2015-02-08 15:25:58 -0600
Processing by Groups::MilestonesController#index as HTML
Parameters: {"state"=>"closed", "group_id"=>"my-group"}
Completed 200 OK in 61ms (Views: 43.9ms | ActiveRecord: 12.2ms)
```

**Comment by Razer6** [comment]
Thanks for the issue report. This issue has already been fixed in newer versions of GitLab. Due to the size of this project and our limited resources we are only able to support the latest stable release as outlined in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#issue-tracker). In order to get this bug fix and enjoy many new features please [upgrade](https://gitlab.com/gitlab-org/gitlab-ce/tree/master/doc/update). If you still experience issues at that time please open a new issue following our issue tracker guidelines found in the [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#issue-tracker-guidelines).
